src/generate.cc

```diff
@@ -980,11 +939,11 @@ static void place_random_door(int y, int x)
 		cave_set_feat(y, x, FEAT_BROKEN);
 	}
 
-	/* Secret doors (200/1000) */
+	/* Secret doors (200/1000) - now locked instead */
 	else if (tmp < 600)
 	{
-		/* Create secret door */
-		place_secret_door(y, x);
```

Comment:
Edit: Deathwatch has granite walls now, but I'm not actually sure it ever had secret doors?

Yup, the two artifacts are hidden behind secret doors. I believe the doors to the stairs down is also a hidden door. (You need pretty searching to detect them, so "Slug-like" and "Coward" and hanging around for quite a few turns.)

Granite is fine.

> (Note: IMO we should change the default colors for permawalls and illusion walls.)

I think it's always been like this; even the secret doors were colored like granite (so different from permawalls).

Regardless, a different issue.